spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/KafkaMessageChannelBinder.java

```diff
@@ -305,11 +341,12 @@ public void onSuccess(SendResult<byte[], byte[]> result) {
 										"Sent to DLQ " + sb.toString());
```

Comment:
There is similar log message not visible here on GH: `"Error sending to DLQ" + sb.toString()`.
Must contains space after `DLQ` word.